### ISSUE TITLE
Docs conflict.

### ISSUE THREAD

**Issue by robin-swift** [Embroidermodder/libembroidery#187]
Hi, I'm back from holiday.

I'm trying to centralise the documentation under one markdown based system, can you look at what I've done and approve me moving the docs here to meld into that system @JoshVarga 

Cheers,
Robin

**Comment by JoshVarga** [comment]
Yeah, I am open to any documentation. I do like the academic approach with tex/pdf. The reason I did the markdown version on libembroidery is I think most of the documentation will be more easily found in an HTML format. If I had to guess, from passed experience most of the documentation will be used by people trying to replicate the reading of the formats in their own projects :)

While you were out I just wanted to throw out a few ideas, the cmake with ctest in parallel with Bazel, and the mkdocs documentation. Those are mostly there as suggestions.
